fleetspeak/src/admin/cli/cli.go

```diff
@@ -84,12 +84,16 @@ func Execute(conn *grpc.ClientConn, args ...string) {
 
 // ListClients prints a list of all clients in the fleetspeak system.
 func ListClients(c sgrpc.AdminClient, args ...string) {
-	if len(args) > 0 {
-		Usage()
-		os.Exit(1)
+	var ids [][]byte
+	for i, arg := range args {
+		id, err := common.StringToClientID(arg)
+		if err != nil {
+			log.Exitf("Unable to convert [%s] (id %d) to client id: %v", arg, i, err)
```

Comment:
Using 'id' to refer to the argument index is potentially ambiguous. Maybe use 'arg' or 'index'?

Reply:
done